src/masoniteorm/schema/platforms/PostgresPlatform.py

```diff
@@ -56,6 +57,7 @@ class PostgresPlatform(Platform):
         "current": " DEFAULT CURRENT_TIMESTAMP",
         "now": " DEFAULT NOW()",
         "null": " DEFAULT NULL",
+        "auto_uuid": " DEFAULT uuid_generate_v4()"
```

Comment:
This needs a test. This is also done through the mixin.

If we are adding this as a valid default we'll need to do the same thing for the other schema classes (mysql, mssql, sqlite)

Reply:
Although it might be handled gracefully already but either way it needs a test to make sure adding a column default of uuid doesn't break anything

Reply:
Yeah I see what you mean.  
The reson I did it like this is the DEFAULT clause is fully escaped ie quoted so the uuid_generate_v4() function cannot be added by the .default() method.

I did not find where the self.default is added to the query as I was hoping to add a .default_raw() which would be used in the same way as .default() but not quoted.
Or adding a parameter to the .default() of raw=False would also work and provide maximum flexibility.

@josephmancuso  WDYT

Reply:
The default values in the dictionary keys are added via the blueprint class. Then the defaults are fetched via that dictionary by the key which adds the dictionary value to the create/alter query